### DISCUSSION TITLE
Displaying only .cue files in the PS folder

### DISCUSSION THREAD

**Discussion by NikitaRUniverse** [OnionUI/Onion#568] [Q&A] [answered]
After the 4.0.2 update, game images are not duplicated in the PS folder, which is good, but this led to the problem that .cue files are not displayed if the game has several bin files, so I want to make the bin files invisible so that only the cue is displayed. Is it possible? Thank you in advance!

**Comment by Aemiii91** [comment, marked as answer]
Yes, we made the wrong decision of indexing `bin` files instead of `cue` files. This will be corrected in the next version.

It was a hard choice because a lot of people are only using single `bin` files for their PS collections. Even some of the romsets out there only contain `bin` files.

But after a [poll on Reddit](https://www.reddit.com/r/MiyooMini/comments/xkziuo/how_do_you_store_your_psx_games_onion_user_onion/), and feedback from the community, we see we need to only index the `cue` files, since these can belong to multiple `bin` files for the same game.

For an immediate fix, edit `Emu/PSX/config.json` in a text editor. In the `extlist`, replace `bin|BIN` with `cue|CUE`.

**Comment by Aemiii91** [comment]
Furthermore, we really recommend converting PS games to `chd` files, which is much smaller in size, but retains the performance and quality.
